src/test/utils.ts

```diff
@@ -61,6 +61,8 @@ export const runAuthenticatedQuery = (
 
 let mergedSchema
 
+// TODO: runQueryMerged could be removed now?
```

Comment:
I guess so. Is it clear how to mock data for stitched links?

Reply:
Yes, there is a test that does this by injecting into the gravQL schema - https://github.com/artsy/metaphysics/blob/f9a68d155ffdbdeb832e72394568d7eea3150dd3/src/schema/me/__tests__/record_artwork_view.test.js#L9-L43